.github/workflows/CI.yaml

```diff
@@ -200,3 +165,27 @@ jobs:
         with:
           name: ${{ matrix.config.name }}-manifest
           path: ${{ matrix.config.folder }}/config/rendered/release.yaml
+
+  component_tests:
+    name: Component Tests
+    needs: prepare_ci_run
+    uses: ./.github/workflows/component-test.yml
+    secrets: inherit
```

Comment:
I don't think we actually need the secrets in all the test pipelines, maybe we can just not pass them on?